<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListDeals</class>
 <widget class="QDialog" name="ListDeals">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1044</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="cb_column">
       <item>
        <property name="text">
         <string>Deal ID</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Contact</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>EMail</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CNIC</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Product ID</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Product Name</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_text"/>
     </item>
     <item>
      <widget class="QPushButton" name="btn_search">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../resources/icons.qrc">
         <normaloff>:/icons/icons/search.svg</normaloff>:/icons/icons/search.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="v_layout">
     <item>
      <widget class="QListWidget" name="listWidget"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../resources/icons.qrc"/>
 </resources>
 <connections/>
</ui>
